doc/cla/individual/tafaRU.md

```diff
@@ -8,4 +8,4 @@ declaration.
 
 Signed,
 
-Alex Comba alex.comba@agilebg.com https://github.com/tafaRU
+Alex Comba alex.comba@agilebg.com (up to 2022-08-26) https://github.com/tafaRU
```

Comment:
Sorry if I was not clear. The `up to <date>` is just a convention we use to say the CLA is no longer valid past this date. You can write it next to the CLA of the users that should no longer commit using this email address (so the ones no longer working for your company). So don't write it next to the emails of the ones still working at agilebg.

Reply:
Ok no problem, I should have fixed.